docs/content/users/topics/phpstorm.md

```diff
@@ -2,96 +2,99 @@
-3. File changes are noticed only by polling, and PHPStorm will complain about this in the lower right, “External file changes sync may be slow”.
-4. Turn off your Windows firewall temporarily. When you have everything working you can turn it back on again.
-Use `ddev start` and `ddev xdebug on`
-5. Click the Xdebug listen button on PHPStorm (the little phone icon) to make it start listening.
-6. Set a breakpoint on or near the first line of your index.php
-7. Visit the project with a web browser or curl. You should get a popup asking for mapping of the host-side files to the in-container files. You’ll want to make sure that `/home/<you>/.../<yourproject>` gets mapped to `/var/www/html`.
+1. Your working project will be on the `/home` partition, so you’ll open it using Windows PhpStorm as `\\wsl$\Ubuntu\home\<username>\...\<projectdir>`.
+2. On some systems and some projects it may take a very long time for PhpStorm to index the files.
+3. File changes are noticed only by polling, and PhpStorm will complain about this in the lower right, “External file changes sync may be slow”.
+4. Temporarily disable your Windows firewall. When you have everything working, you can turn it back on again.
+5. Use `ddev start` and `ddev xdebug on`.
+6. Click the Xdebug listen button on PhpStorm (the little phone icon) to make it start listening.
+7. Set a breakpoint on or near the first line of your `index.php`.
+8. Visit the project with a web browser or cURL. You should get a popup asking for mapping of the host-side files to the in-container files. Make sure `/home/<you>/.../<yourproject>` is mapped to `/var/www/html`.
 
 Debugging should be working. You can step through your code, set breakpoints, view variables, etc.
 
-Set the PHPStorm terminal path (Settings→Tools→Terminal→Shell Path) to `C:\Windows\System32\wsl.exe`. That way when you use the terminal Window in WSL2 it’s using the bash shell in WSL2.
+Set the PhpStorm terminal path (*Settings* → *Tools* → *Terminal* → *Shell Path*) to `C:\Windows\System32\wsl.exe`. That way when you use the terminal Window in WSL2 it’s using the Bash shell in WSL2.
 
-### PHPStorm inside WSL2 in Linux
+### PhpStorm Inside WSL2 in Linux
 
-1. On Windows 11 you don't need to install an X11 server, because [WSLg](https://github.com/microsoft/wslg) is included by default. On older Windows 10, Install X410 from the Microsoft Store, launch it, configure in the system tray with “Windowed Apps”, “Allow public access”, “DPI Scaling”→”High quality”. Obviously you can any other X11 server, but this is the one I’ve used.
+1. On Windows 11 you don’t need to install an X11 server, because [WSLg](https://github.com/microsoft/wslg) is included by default. On older Windows 10, Install X410 from the Microsoft Store, launch it, configure in the system tray with “Windowed Apps”, “Allow public access”, “DPI Scaling” → ”High quality”. You can any other X11 server, this is just the one we’ve used.
```

Comment:
```suggestion
1. On Windows 11 you don’t need to install an X11 server, because [WSLg](https://github.com/microsoft/wslg) is included by default. On older Windows 10, Install X410 from the Microsoft Store, launch it, configure in the system tray with “Windowed Apps”, “Allow public access”, “DPI Scaling” → ”High quality”. You can any other X11 server, this is just one possibility.
```